app/code/Magento/Sales/view/base/ui_component/customer_form.xml

```diff
@@ -6,10 +6,10 @@
  */
 -->
 <form xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Ui:etc/ui_configuration.xsd">
-    <htmlContent name="cart_content">
-        <block class="Magento\Customer\Block\Adminhtml\Edit\Tab\Carts" name="cart"/>
-    </htmlContent>
     <htmlContent name="orders_content">
         <block acl="Magento_Sales::actions_view" class="Magento\Sales\Block\Adminhtml\CustomerOrdersTab" name="orders" />
     </htmlContent>
+    <htmlContent name="cart_content">
+        <block class="Magento\Sales\Block\Adminhtml\ShoppingCartsTab" name="cart"/>
```

Comment:
I guess it should be unders ACL, similar to "view orders". Could you update it?

Reply:
Done. Please check.